composer/datasets/coco.py

```diff
@@ -23,7 +23,7 @@ def _isArrayLike(obj):
 class COCODatasetHparams(DatasetHparams):
     """Defines an instance of COCO Dataset.
 
-    Parameters:
+    Args:
```

Comment:
Also the following public objects are missing docstrings:
- [ ] `dboxes300_coco`
- [ ] `split_dict_fn` 
- [ ] COCODetection